### ISSUE TITLE
Devilsaurs do not patrol

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#6242]
> Devilsaurs do not patrol

.go xyz -6622.660000 -989.547000 -271.236000 1 5.892780

**Comment by Hatson22** [comment]
This is correct for some of them. Specifically creature ID 6500. This seems to be the only one that isn't patrolling. I think all the other spawns are. 
For reference this is the Tyrant Devilsaur spawn on the east side in the middle, in The Marshlands sub-section.

**Comment by Hatson22** [comment]
Duplicate of #625
